### ISSUE TITLE
[Item]The Lightning Capacitor

### ISSUE THREAD

**Issue by ircfelix** [Atlantiss/NetherwingBugtracker#3136]
**Description**:
Currently the trinket The Lightning Capacitor is not behaving as it should. This is somehow crucial, since this trinket was considered to be the best caster trinket in TBC for a very long time.

**Current behaviour**:
1. Not every crit gives you a charge (tested with Arcane Explosion at the Shattrath dummies).
2. The bolt fired by this item is not scaling with spell power.
3. The trinket has an internal cooldown.

**Expected behaviour**: 

1. Every crit should give you a charge. Also when doing multiple crits, e.g. from aoe, you should get n (number of crits in one spell) charges from that one spell. The only spell that does NOT give you charges are the criticals from the bolts the trinket fires.

>  the crits of the bolt do not produce a charge, but any spell-like critical hit inflicted by your character will produce a charge, e. g. Molten Armor, Lightning Shield, etc. and of course single target direct damaging spells and AoE spells.

_Source for the multi crit thing: http://wowwiki.wikia.com/wiki/The_Lightning_Capacitor?oldid=1553723_
Citation:
> Electrical charges CAN be gained on every critical strike except for the first 2.5 seconds after a bolt goes off. This means it's possible to get 3 crits at once (for example, from an AoE spell hitting multiple targets) and fire off a single bolt. However, any further crits will not produce charges until after 2.5 seconds.

2. Until patch 2.1 the item should be scaling with spell power

> the bolt emitted by this trinket is not affected by +Spell damage anymore (since Patch 2.1)

3. Until patch 2.3 it should not have the 2.5s internal cooldown

> This item was changed in patch 2.3. It now has a 2.5 second cooldown after each lightning bolt.

 _Source for all 3 bugs: http://wowwiki.wikia.com/wiki/The_Lightning_Capacitor?oldid=1070605_

**Server Revision**: 
2370

**Comment by Easyfraggy** [comment]
We are still not decided on reverting trinket ICD's and proc rates to 2.0 state - as these were often nerfed because they were extremely OP like DST and they kind of fall under the 2.4.3 mechanics umbrella which we generally preserve. Additionally the point of 2.0 gear is to make players weaker - this kind of goes against that also.

We are well aware that LCap and all other trinkets are currently using their 2.4.3 proc rates and mechanics.

**Comment by Evolvee** [comment]
https://github.com/Atlantiss/AtlantissTBCBT/issues/4231

**Comment by ircfelix** [comment]
I personally do not care about points 2 or 3, however, I don't understand why you make everything with 2.0 stats, but allow things like Curse of Elements or unnerfed 2.4.3 talents, but start cherry-picking when it comes to a hand full of items. Those few special items won't make a game breaking difference, unless you stack only players with those OP items in one raid.

But back to topic: Regardless of how you decide with number 2 and 3, number 1 is bugged.
If you AOE multiple targets (let's say once every 5 seconds) and score a crit, often you will get no charge at all, which is not how this trinket is supposed to work.

Also with the 2,5s internal CD in place right now, it does not really matter if you get a charge once per spell or for every crit. Let me explain you why. If you get 3 crits within the 2,5s, you get one bolt. Assuming you cast something like arcane explosion, which has a 1,5s cooldown and you don't get multiple charges for multiple crits, you have to cast 3 arcane explosions. Now going further and assuming one crit per aoe, you get one bolt each 4,5s in the best case. That is ~700 damage per 2,5s vs 700 dmg vs 4,5s. That is 280 bolt dps if you get multiple charges for multiple crits every 2,5s vs 156 dps if you only get one charge per multiple crits. You can say that this is a stunning 124 dps difference, which is true, however, it is 124 dps for aoeing **trash mobs**, in 90% of the cases. So I really ask myself if it is reasonable to nerf this trinket in an unblizzlike way just to make things harder. If you nerf it too much, people will just use different trinkets, because it's not worth it any more.

**Comment by Easyfraggy** [comment]
Whatever you do you have to set a line somewhere.

Wether trinket ICD's and mechanics fall under Mechanics 2.4.3 or Items 2.0 is the real question. For me it would be mechanics. Just like we aren't changing talents expertise and spells.

**Comment by ircfelix** [comment]
After sleeping a night over this, I have come to the conclusion that only point 2 makes a difference for single target dps and point 1 (except for the bug that you sometimes get no charges at all from aoe crits) and 3 will not influence single target dps, simply because you can't produce 3 crits in under 2,5s with single target spells. The best you can do with single target spells is 3 crits in 4,5s but having 3 crits in a row is extremely unrealistic.

For point 2 I have to say that I guess the spell damage scaling should not be that high, since the bolt goes off in an instant. In Vanilla wow instant casts were modified with a 1,5/3,5=0,428 spell damage coefficient. If this is also the case here, you would need to have 1000 (nature) spell damage to increase the bolt damage by 428 before any partial resists. If you crit 3 times in a row in 4,5 seconds, that's a ~95dps increase, however, it is more likely that it takes you at least something like 15 seconds to score 3 crits. The 428 bonus damage from spell damage scaling over 15 seconds would only be a ~28 dps increase. 
If you compare TLC with Icon of the Silver Crescent, for example, the best IotSC can give you is a ~20,5 dps increase from the passive spell damage only (for Arcane blast with 1,5s cast time the calculation is 43*(5/7)/1,5, where 5/7 is the spell damage coefficient of AB).

You can conclude that removing the spell damage scaling of the bolt for single target dps to negate a 8 dps difference (in the best case) isn't really worth it.

**Comment by Saben65** [comment]
The issue with not getting a charge with AOE seems to be tied to this issue https://github.com/Atlantiss/NetherwingBugtracker/issues/3477.

**Comment by ShadyWins** [comment]
**Please have another look at this item, as it's not the way it was pre patch 2.3.**

_"This item was changed in Patch 2.3. It now has a 2.5 second cooldown after each lightning bolt."_ 
https://wowwiki.fandom.com/wiki/The_Lightning_Capacitor?oldid=1553723

Due to the fact Netherwing is a progressive TBC realm currently emulating patch 2.0 The Lightning Capacitator should be in the state, that it was prior to the 2.3 nerf. 

As of right now, this item is how it was AFTER the 2.3 nerf.

The current state of The Lightning Capacitator on Netherwing is not how it's supposed to be, in regards to the current patch the server is emulating.

Furthermore, this is a trinket - an item - and all items (that I know of) have been rolled back to their 2.0 state, therefore it leads me to believe, that this must be a mistake.

Best regards a very concerned elemental shaman, who just realized his BiS trinket is not as intended in regards to the patch we're playing.

I sincerely hope you'll take another look at this, and take into account that this Trinket is an item, and not a spell.

**Server Revision:** 2819

**Comment by Easyfraggy** [comment]
Like I've said earlier trinket mechanics are a problem because they were changed a lot through the patches and some were just straight up broken in early patches - like for example DST. For now it was decided not to touch these and leave them 2.4.3. Unless Wolf decides to revert all trinkets to 2.0 it's not gonna change for 1 trinket but all if it does.

**Comment by Chandalennn** [comment]
Even if you want it in the later patch states, it's still not working correctly. "Electicral Charge cannot be gained more often than once every 2.5s sec" sounds like it stops you from getting another charge if you got any crits in the last 2.5s, and that's how it works on Netherwing, but it's not how it worked in retail. It should only stop you from gaining charges only after firing the lightning bolt, not after every charge gained. 

See: https://wowwiki.fandom.com/wiki/The_Lightning_Capacitor?oldid=1553723
`The tooltip is somewhat misleading regarding the 2.5 second internal cooldown. Electrical charges CAN be gained on every critical strike except for the first 2.5 seconds after a bolt goes off. This means it's possible to get 3 crits at once (for example, from an AoE spell hitting multiple targets) and fire off a single bolt. However, any further crits will not produce charges until after 2.5 seconds. This effectively limits the bolts to one per 2.5 seconds. A more correct tooltip might read: "Electrical Charge cannot be gained for 2.5 seconds after a Lightning Bolt is fired.`

Also see:  https://www.wowhead.com/item=28785/the-lightning-capacitor#comments
The comment from FinnrSlayer posted in 2.4.1, in addition to many others
`This trinket is underestimated is many ways. The 2.5 Second Cool down only turns on after you get the lightning bolt from it. On my Elemental Shaman i gotten 3 crits with Chain Lightning and it procs one extra lightning bolt from it. Also the proc crits like mad when you have more spell crit!`


Maybe this is different enough than what ircfelix started with that it should be a separate issue, let me know. It's similar to his point 1, but definitely different since this is about it still having the cooldown after a bolt but not between charges, whereas he wanted it to never have a cooldown as it was in early patches.

**Comment by Easyfraggy** [comment]
@Chandalennn that's a global issue for all abilities/procs. Atm killing blows don't proc anything, talents/trinkets etc.

**Comment by dorkyo** [comment]
Yeah can't lie the ele shaman fantasy basically killed by this nerf, as strong as elemental remains in certain situations our single target dps falls behind immensely without incredible RNG at our side. With an item nearly built around a class and spec (You could tag it Classes: Shaman and no one would question it), I don't understand how you can continue you let this float by without at least an attempted test at a revision of the trinket. 

Pretty disheartening to have banked what's supposed to be BiS for tiers, for a blue trinket since the day I obtained it because of pick-and-choose nerfs and refusal to test :\

**Comment by Easyfraggy** [comment]
You do realize that arcane mage would be the broken spec with this item reverted right? Because you'd also have to revert the haste meta gem to be consistent aswell as DST ICD etc. Ele shamans do perfectly fine like they are - noone is going to bench an ele shaman from raid because of how much dps they do. They could do 600 or less and still have a spot.

**Comment by artishoka1** [comment]
I have read several of your statements regarding balance and have to say your arguments are quite invalid. If you continue to produce arguments without evidence of your actions, you should remove this from your website - "This way players will be able to experience the true Blizzlike progression that will start with The Burning Crusade". You have to realize that your decisions to nerf some items and buff others are neither "true" or "Blizzlike". At this moment Netherwing should not be distinguished as Blizzlike experience, but rather a fun server with randomized values.

**Comment by Easyfraggy** [comment]
You do realize by that definition every server is a funserver. Every server runs content from patch 1.0.3/2.0.3 etc with the latest patch of the expansion and do CUSTOM CHANGES to gear/difficulty/rates etc. So every server is a funserver - congrats. You are just so incredibly stuck only being able to see your own nose that you can't see the big picture.

It has been said from the start that mechanics are 2.4.3 which includes these trinket mechanics player spells etc. Only PvE and item stats are 2.0 and tuned to mimic the difficulty of early prepatch encounters.

You're the one who wants to cherrypick this trinket to be incredibly OP without thinking about the ramifications.

"Joined 17 hours ago" made 1 invalid bugreport. Very helpful to the success of server development.

**Comment by artishoka1** [comment]
I have not read anything like that, stating that item mechanics will be according to patch 2.4.3. I guess you just made that up. As evidence this is what is stated on Netherwing site:

Q: What is 2.0 itemization?

It is a form of a blizzlike pre-nerf state that will be present on Netherwing. Many items from Heroic dungeons and T4 and T5 content were somewhat weaker in patch 2.0 . We will be bringing back these weaker stats on items in order to try to increase the overall difficulty of our content. 

It is never mentioned that trinket or item mechanics will be on patch 2.4.3 or they should be the same as patch 2.4.3. 

And to answer the accusation regarding cherry picking, you are the one stating that items such as Annihilator or Nightfall were not used in raids, based on the fact there are no videos of it, while there is ton of evidence regarding the optimization of DPS  in raiding guilds from Vanilla and TBC through itemization with these items all over the internet.

Same is the condition with Sextant of Unstable Currents - the best trinket in T5 content, now it is completely useless since you are using the values from patch 2.4.3 instead of 2.0.3 where the procrate of that trinket was 50%. Now this trinket is comparable to Vengeance of the Illidari, a level 61 trinket from quest.

The development team can do whatever they want with the server and with the server values, but it is them, who made this bugtracker to report and discuss bugs, such as it is with the items already stated. However, your attitude is clear example, why most players do not participate in the bug tracker, since they will be hit with "WE DECIDED" argument.

**Comment by ircfelix** [comment]
> 
> 
> I have not read anything like that, stating that item mechanics will be according to patch 2.4.3. I guess you just made that up. As evidence this is what is stated on Netherwing site:
> 
> Q: What is 2.0 itemization?
> 
> It is a form of a blizzlike pre-nerf state that will be present on Netherwing. Many items from Heroic dungeons and T4 and T5 content were somewhat weaker in patch 2.0 . We will be bringing back these weaker stats on items in order to try to increase the overall difficulty of our content.
> 
> It is never mentioned that trinket or item mechanics will be on patch 2.4.3 or they should be the same as patch 2.4.3.
> 
> And to answer the accusation regarding cherry picking, you are the one stating that items such as Annihilator or Nightfall were not used in raids, based on the fact there are no videos of it, while there is ton of evidence regarding the optimization of DPS in raiding guilds from Vanilla and TBC through itemization with these items all over the internet.
> 
> Same is the condition with Sextant of Unstable Currents - the best trinket in T5 content, now it is completely useless since you are using the values from patch 2.4.3 instead of 2.0.3 where the procrate of that trinket was 50%. Now this trinket is comparable to Vengeance of the Illidari, a level 61 trinket from quest.
> 
> The development team can do whatever they want with the server and with the server values, but it is them, who made this bugtracker to report and discuss bugs, such as it is with the items already stated. However, your attitude is clear example, why most players do not participate in the bug tracker, since they will be hit with "WE DECIDED" argument.

i am no dev, but my bet is this won't ever be fixed since it would make a hand full of players stronger.

**Comment by Suzaru** [comment]
Just like the Mystical Skyfire Diamond. Is this going to get fixed? Vital for arcane mages :(

**Comment by Manaci** [comment]
Irrelevant now

**Comment by Manaci** [comment]
Reopened for future usage

**Comment by Godmatik** [comment]
Rev 4015 - Issue 1 must've been resolved a while ago - every crit is giving a charge

Issues 2 and 3 will not be implemented, trinkets will be in their 2.4.3 state.

Closing
